src/components/about/AboutHeaderImage.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Container, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(theme => {
+  return {
+    imageContainer: image => ({
+      width: "100%",
+      height: 373,
+      backgroundImage: image.url,
+      backgroundPosition: "center",
+      backgroundSize: "cover",
+      marginBottom: theme.spacing(2.5),
+      display: "flex",
+      flexDirection: "column",
+      justifyContent: "center",
+      textAlign: "center"
+    }),
+    headlineText: {
+      color: "white",
+      border: "5px solid white",
+      display: "inline-block",
+      margin: "0 auto",
+      paddingTop: "3%",
+      paddingBottom: "3%",
+      paddingRight: "5%",
+      paddingLeft: "5%"
+    }
+  };
+});
+
+export default function AboutHeaderImage({ image }) {
```

Comment:
I was a little surprised that we were passing this in as a prop instead of having the `AboutHeaderImage` "know" about which background image to use. Why is it passed in?

Reply:
You're right, it doesn't really help. I changed it.